### ISSUE TITLE
Human GO slim

### ISSUE THREAD

**Issue by goodb** [geneontology/helpdesk#303]
Is there a public GO subset for Human ?  I was surprised not to find one at http://geneontology.org/docs/download-ontology/ .  I imagine this might be a common request - e.g. see https://www.biostars.org/p/329226/

If not, why?
thanks!

p.s. http://amigo1.geneontology.org/cgi-bin/amigo/slimmer seems to be dead.

**Comment by suzialeksander** [comment]
Hey @goodb, 

For users looking for a human Slim, we usually refer them to the Generic Slim: https://geneontology.github.io/docs/download-ontology/#download-go-subsets.  If for some reason the Generic Slim doesn't particularly fit, we can help make a custom slim. 

For the AmiGO slimmer, the URL makes me suspicious that this was closed down quite a while ago, but @kltm can confirm- where did you find the link? https://www.yeastgenome.org/goSlimMapper and https://go.princeton.edu/GOTermMapper/GOTermMapper_help.shtml (functionally the same as each other) are the current recommended tools, although I can't remember exactly what the AmiGO tool did.

**Comment by kltm** [comment]
@suzialeksander Indeed, the slimmer in AmiGO was shutdown back in the AmiGO 1.x days.

**Comment by goodb** [comment]
The link to the AmiGO slimmer was on the bottom of:
https://geneontology.github.io/docs/go-subset-guide/ 

The generic slim isn't a good fit.  I was looking for something that eliminated terms without any associated human genes (following inference).  (e.g. no plant or bacteria-specific terms)  I made one (thank you robot) that has on the order of 20,000 terms - quite a different thing than the very small generic slim.  

It seems strange to have slims built specifically for several other important organisms but not for human.  For the public, having a GO 'taxon slim' for each important creature seems useful.

**Comment by suzialeksander** [comment]
Adding to this ticket as a reminder: Gene Godbold (Signature Science) presented a use case for a pathogen-slim at the GOC meeting

**Comment by ValWood** [comment]
@ValWood tagging myself on this for future ref.
